files/templates/scrapeconfigs/_node_exporter_namespace.yaml

```diff
@@ -1,7 +1,3 @@
 [[- define "_node_exporter_namespace" -]]
-[[- if eq .ClusterType "management_cluster" -]]
-monitoring
-[[- else -]]
-kube-system
-[[- end -]]
+monitoring|kube-system
```

Comment:
I'd keep the `if` still because customers can deploy their own `node-exporter` to their own `monitoring` namespace.

Reply:
wouldn't that be covered already by the regex?

Reply:
That's the problem. We don't want to scrape customer's node-exporter.